src/main/java/CliDisplay.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Scanner;
+
+public class CliDisplay implements Display {
+
+    public void sendMessageToDisplay(String message) {
+       System.out.print(message);
+    }
+
+    public String getResponse(String request) {
+        Scanner userInput = new Scanner(System.in);
+        System.out.print(request);
+        return userInput.nextLine();
+    }
```

Comment:
I could agree with this, but splitting it out will mean the calling class will have to make two method calls to CLIDisplay which can be a bit messy.  Its a tradeoff! 
eg
sendMessage("Next move please:")
nextMove = getResponse()